resources/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z $TF_RUNNER_VERSION ]]; then
```

Comment:
Bracketing for consistency `${TF_RUNNER_VERSION}`

Reply:
Updated in [ecf9fa1](https://github.com/companieshouse/infrastructure-terraform-runner/pull/31/commits/ecf9fa152110593d1ec941d81f45221f14714759) 👍